include/odp/api/spec/pool.h

```diff
@@ -214,6 +221,17 @@ typedef struct odp_pool_param_t {
 			    defined by pool capability pkt.max_uarea_size.
 			    Specify as 0 if no user area is needed. */
 			uint32_t uarea_size;
+
+			/** Minimum Headroom size in bytes. Minimum number of
+			    bytes of space preceding packet data to reserve for
+			    use as headroom. Each newly allocated packet from
+			    the pool must have at least this much headroom.
+			    Must not exceed the implementation defined
+			    'max_headroom' in odp_pool_capability().
+			    @note If application does not require headroom it
+			    can configure 'min_headroom' as zero and
+			    implementation may or may not allocate headroom. */
+			uint32_t min_headroom;
```

Comment:
The parameter name can be just "headroom".

Text can be compressed a bit and made inline with documentation of the other params:

"Minimum headroom size in bytes. Each newly allocated 
packet from the pool will have at least this much headroom. 
The maximum value is defined by pool capability pkt.max_headroom.
Use zero if headroom is not needed."

Reply:
I prefer naming it as min_headroom is coz the value given by application is not directly configured and implementation may roundup the value and hence the name "min_headroom"

Reply:
Sorry, forgot to mention the rationale for "headroom" vs "min_headroom". Plain "headroom" is inline with the other parameter names in the struct (num/len/max_len/seg_len/uarea_size). Those describe in documentation text how implementation may round up things, but the variable names are not min_len, min_seg_len, min_uarea_size, etc 

For example, seg_len spec is very close to headroom spec. Application requires at least 'seg_len' bytes but implementation is free to round up:
"Minimum number of packet data bytes that are stored
in the first segment of a packet. The maximum value
is defined by pool capability pkt.max_seg_len.
Use 0 for default. "
uint32_t seg_len;

So, please rename to "headroom".